lib/common/service/character_service.dart

```diff
@@ -10,4 +10,12 @@ class CharactersService {
   final CharactersApi _charactersApi;
 
   Future<Characters> getCharacters() => _charactersApi.getCharacters();
+
+  Future<void> addCharacter(Character newCharacter) async {
+    try {
+      await _charactersApi.addCharacter(newCharacter);
+    } catch (e) {
+      throw Exception('Error during adding the character: $e');
```

Comment:
add todo here, maybe we'll add it later
but now I don't think that it's necessary
```suggestion
      // TODO emit failure state
      throw Exception('Error during adding the character: $e');
```

Reply:
oh no, I thought it's bloc class then we don't need try catch at all

Reply:
Ok
Clear. I have deleted it.
As I have understood this one is as interface? Is it correct?